wagtail/admin/widgets/tags.py

```diff
@@ -39,21 +42,44 @@ def get_context(self, name, value, attrs):
         else:
             free_tagging = self.free_tagging
 
-        tag_spaces_allowed = getattr(settings, "TAG_SPACES_ALLOWED", True)
+        if hasattr(settings, "TAG_SPACES_ALLOWED"):
+            warn(
+                "The setting 'TAG_SPACES_ALLOWED' is deprecated. "
+                "Please use 'WAGTAIL_TAG_SPACES_ALLOWED' instead.",
+                RemovedInWagtail70Warning,
+            )
+
+        tag_spaces_allowed = getattr(
+            settings,
+            "WAGTAIL_TAG_SPACES_ALLOWED",  # Make sure this matches your override setting name
```

Comment:
This comment doesn't seem to provide any value

```suggestion
            "WAGTAIL_TAG_SPACES_ALLOWED",
```